protocol/communities/request_to_join.go

```diff
@@ -57,3 +62,18 @@ func (r *RequestToJoin) InitFromSyncProtobuf(proto *protobuf.SyncCommunityReques
 func (r *RequestToJoin) Empty() bool {
 	return len(r.ID)+len(r.PublicKey)+int(r.Clock)+len(r.ENSName)+len(r.ChatID)+len(r.CommunityID)+int(r.State) == 0
 }
+
+func AddTimeoutToRequestToJoinClock(clock uint64) (uint64, error) {
+	requestToJoinClock, err := strconv.ParseInt(fmt.Sprint(clock), 10, 64)
+	if err != nil {
+		return 0, err
+	}
+
+	// Adding 7 days to request
+	// requestTimeOutClock := uint64(time.Unix(requestToJoinClock, 0).AddDate(0, 0, 7).Unix())
+
+	// For testing on client - setting 5 minutes as timeout
+	requestTimeOutClock := uint64(time.Unix(requestToJoinClock, 0).Add(time.Minute * 5).Unix())
```

Comment:
You forgot to switch the comments

Reply:
The Mobile PR is open, and we need this 5-minute timeout (requested by the QA team) until the testing is completed.

I will remove this 5-minute timeout and will replace/uncomment the required seven days timeout once the testing is completed.